rules/REQUEST-903.9006-XENFORO-EXCLUSION-RULES.conf

```diff
@@ -148,7 +149,8 @@ SecRule REQUEST_FILENAME "@streq /inline-mod/" \
 
 # Warn member
 # POST /xf/members/name.12345/warn
-SecRule REQUEST_FILENAME "@rx /members/\*\.\d+/warn$" \
+# POST /xf/posts/12345/warn
+SecRule REQUEST_FILENAME "@rx /(?:members/\*\.\d+|posts/\d+)/warn$" \
```

Comment:
Is this really meant to match e.g. the string `members/*.1234/warn` literally? 
In other words, is the escaping of `*` and `.` on purpose?

Reply:
Well spotted, the `*` should not have been escaped.